src/components/buttons/Button.spec.ts

```diff
@@ -2,35 +2,39 @@ import { describe, expect, it } from 'vitest';
 import { mount } from '@vue/test-utils';
 import Button from '@/components/buttons/Button.vue';
 
+const createWrapper = (props: Record<string, any> = {}) =>
+  mount(Button, {
+    props,
```

Comment:
Just out of curiosity, are props as a type a record too? or do they have a more specific type here?

Reply:
From the mount function, it can be more specific type, but unfortunately it's not exported 
<img width="654" alt="image" src="https://github.com/rotki/ui-library/assets/26648140/8019dd85-8c73-43f1-a9f3-9b88a2db8cad">

Reply:
So ho way to infer it somehow then 😢